rust/main/agents/relayer/src/settings/matching_list.rs

```diff
@@ -236,6 +267,8 @@ pub struct ListElement {
     destination_domain: Filter<u32>,
     #[serde(default, rename = "recipientaddress")]
     recipient_address: Filter<H256>,
+    #[serde(default, rename = "regex")]
+    body: Option<RegexWrapper>,
```

Comment:
A little bit confusing name of the field `body` with rename `regex`. Possibly, `body_regex` and corresponding `bodyregex` are clearer.

Reply:
ah, good call. I'll change it to `body_regex`